payroll-processor-functions/src/payroll-processor-functions/Features/Payrolls/PayrollEntity.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace PayrollProcessor.Functions.Features.Payrolls
+{
+    public class PayrollEntity : TableEntity
+    {
+        public DateTimeOffset CheckDate { get; set; }
+        public Guid EmployeeId { get; set; }
+        public double GrossPayroll { get; set; }
+        public string PayrollPeriod { get; set; } = "";
+        public string EmployeeDepartment { get; set; } = "";
+
+        public static class Map
```

Comment:
This is a convenient convention for mapping between entity/domain types.

I added it as a class on the entity type to make it easy to access the write mapper-per-type.

Reply:
I like this over using AutoMapper for our domain, since we should only have explicit conversions between types